custom_components/tuya_local/devices/xoca_dac2121c_energy_meter.yaml

```diff
@@ -0,0 +1,158 @@
+name: XOCA DAC2121C BI Energy Meter
```

Comment:
The branding should be under the products section below. This top level name should be generic in case of rebranded devices (as is common with Tuya devices sold in different parts of the world).

Reply:
I own rebranded one which reports itself in Tuya as XOCA DAC2121C BI. Have no idea what would generic look like.

Reply:
Generic would be "Energy meter"